src/config.map.json

```diff
@@ -39,33 +54,15 @@
           ]
         },
         "fill-opacity": 0.7
-      },
-      "filter": [
-        "==",
-        "feature_type",
-        "aggregation"
-      ]
+      }
     },
     {
-      "id": "cluster-heat",
+      "id": "cluster_heat",
       "type": "heatmap",
-      "source": "data_source",
+      "source": "cluster-cluster_heatmap",
```

Comment:
It's look like a bit redundant `cluster-cluster...`

Reply:
I will treat it in the next PR